composer.json

```diff
@@ -10,6 +10,12 @@
   ],
   "homepage": "https://github.com/leeovery/wordpress-to-laravel",
   "license": "MIT",
+  "repositories": [
```

Comment:
Can you remove this section please. Cant merge in the PR with it :)

Reply:
I've removed this and pushed to my develop branch. 
Do I need to do anything else or you can merge it? 
Sorry I'm new to this PR workflow process :)
Thank you